services/app-web/src/pages/StateSubmission/RateDetails/RateDetails.test.tsx

```diff
@@ -541,7 +541,7 @@ describe('RateDetails', () => {
             expect(imageElFile2).toHaveClass('is-loading')
 
             // click continue while file 2 still loading
-            continueButton.click()
+            fireEvent.click(continueButton)
```

Comment:
when do we want to use fireEvent vs. userEvent?

Reply:
I buried this too far in the summary, but fireEvent is still synchronous, so in a case like this, where we're trying to test a transitional state, it can be useful.